pkg/sloop/queries/respayloadquery_test.go

```diff
@@ -182,3 +182,33 @@ func Test_removeDupePayloads_twoTheSame_returnsFirstOne(t *testing.T) {
 	ret := removeDupePayloads(input)
 	assert.Equal(t, expected, ret)
 }
+
+func Test_GetResPayload_True_HasSamePrefix(t *testing.T) {
+	untyped.TestHookSetPartitionDuration(time.Hour)
+	partitionId := untyped.GetPartitionId(someTs)
+	values := helper_get_params()
+	expectedKind := "someKind"
+	expectedNS := "someNamespace"
+	expectedName := "someName-11"
+	values[KindParam] = []string{expectedKind}
+	values[NamespaceParam] = []string{expectedNS}
+	values[NameParam] = []string{expectedName}
+
+	var keys []string
+	keys = append(keys, typed.NewWatchTableKey(partitionId, expectedKind, expectedNS, expectedName, someTs).String())
+	keys = append(keys, typed.NewWatchTableKey(partitionId, "someKind", "someNamespace", "someName-1", someTs).String())
```

Comment:
please use named constant instead of literals: "someKind", "someNamespace". Just use the literals for the new names